zproject/default_settings.py

```diff
@@ -310,14 +310,6 @@
 # Two factor authentication is not yet implementation-complete
 TWO_FACTOR_AUTHENTICATION_ENABLED = False
 
-# This is used to send all hotspots for convenient manual testing
-# in development mode.
-ALWAYS_SEND_ALL_HOTSPOTS = False
-
-# The new user tutorial is enabled by default, but can be disabled for
-# self-hosters who want to disable the tutorial entirely on their system.
-TUTORIAL_ENABLED = True
```

Comment:
We no longer need to provide `TUTORIAL_ENABLED` setting, right ? -- I think notices are less louder than hotspots.